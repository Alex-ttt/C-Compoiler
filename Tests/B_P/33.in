struct point{
	float x;
	float y;
	float z;
};
int func(point *a);

void main(){
 	point *h;
	func(h);
}
