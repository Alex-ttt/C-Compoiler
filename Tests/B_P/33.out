 Symbols Table:
   struct point
          x float
          y float
          z float
   func  int function func(
          a pointer to struct point
  )
   main  void function main(
  )
    {
      Statements:
      func
      (
        h
      )
      Symbols Table:
              h pointer to struct point
    }
